.stylelintrc.js

```diff
@@ -17,34 +17,37 @@ module.exports = {
   reportNeedlessDisables: true,
   reportInvalidScopeDisables: true,
   rules: {
-    'order/order': ['custom-properties', 'declarations', 'rules'],
-    'no-invalid-position-at-import-rule': null,
-    'no-descending-specificity': null,
-    'declaration-no-important': [
-      true,
-      {
-        severity: 'warning',
-      },
-    ],
+    // Turning this off in favor of 'at-rule-allowed-list'
     'at-rule-no-unknown': null,
-    'function-no-unknown': [
-      true,
-      {
-        ignoreFunctions: ['percentage', 'calc', 'math.div'],
-      },
-    ],
-    'selector-pseudo-class-case': 'lower',
-    'order/properties-alphabetical-order': true,
-    'no-empty-first-line': true,
-    linebreaks: 'unix',
     'at-rule-allowed-list': [
-      ['function', 'if', 'use', 'media', 'font-face', 'import', 'extend'],
-      {
-        severity: 'warning',
-      },
+      // Are we intentionally leaving out 'include'?
```

Comment:
Yeah warnings are good for includes, gotta move to use eventually

Reply:
Do you mean import? I can remove import from this list so we get warnings for those, but maybe we should wait until we're ready to migrate to `@use` because it's just noise until we're ready.